azavea/fb-gender-survey-dashboard
# Basic application workflow sequence

## Overview

A basic workflow for selecting regions, questions, and indexing survey response data. The QuestionSelector screen in particular has some complex interrogating of questions to determine what to render. It's clear that we need to develop some heuristics in the questions to determine how best to display them. Some are developed and added here, in a rather rudimentary form (I tried to comment excessively in this case), but more should be expected.

Connects #7 

### Notes

This only includes region data, as it's all we've received but the country data should perform similarly. The config and data generated and checked into this branch was created in a WIP branch that will be updated when we get final datasets.


## Testing Instructions

 * Select a region or two, and a question or two. 
 * Confirm output on /visualization includes indexed survey data for the question
